### DISCUSSION TITLE
Rotating path

### DISCUSSION THREAD

**Discussion by hosmelq** [steveruizok/perfect-freehand#43] [General]
First of all thank you very much for this tool.

I would like to know if there is a way to rotate the path to a specific angle before passing it to the `getStroke` function.

This is for a signature application but I have to be able to rotate the paths.

Thank You!

**Comment by steveruizok** [comment]
Yes, you would need to rotate each point around the common center of all points. Check out the @tldraw/vec package for some helpers with rotating points!
